src/dynapi/SDL_dynapi.h

```diff
@@ -51,8 +51,12 @@
 #define SDL_DYNAMIC_API 0
 #elif defined(SDL_BUILDING_WINRT) && SDL_BUILDING_WINRT  /* probably not useful on WinRT, given current .dll loading restrictions */
 #define SDL_DYNAMIC_API 0
+#elif defined(__PS2__) && __PS2__
+#define SDL_DYNAMIC_API 0
 #elif defined(__PSP__) && __PSP__
 #define SDL_DYNAMIC_API 0
+#elif defined(_EE) /* probably not useful on PS2 */
```

Comment:
What is _EE here?

Reply:
[Emotion Engine](https://en.wikipedia.org/wiki/Emotion_Engine), the CPU used by Sony in PS2.

Reply:
EE is the cpu, and DVP is the vector processor.
ps2dev (=the organization that maintains the ps2 toolchain) provides a C compiler that targets EE and an assembler for DVP.

Reply:
Right, but we've already checked __PS2__ above... ?

Reply:
yes, I need to remove it and use just `__PS2__`